src/en/wizden-staff/maintainer/review-procedure.md

```diff
@@ -1,54 +1,181 @@
+All PRs must be triaged, as per the [triage procedure](triage-procedure.md) guidelines. 
+If a PR is not triaged, this should be done at the same time as the PR is initially considered for review.
 
-- If a Maintainer Discussion thread has been inactive for a week with no new messages, the Discord bot will tag the PR owner and prompt for a resolution. What this resolution is depends on the discussion, but Maintainers are encouraged to arrive at a compromise to either give the PR conceptual approval, agree to close the PR, or in rare cases ask the PR to be set to Draft in case a resolution can not be reached for some unique reason related to the PR. Other actions may also be taken. In the case the Discord bot fails, Maintainers are encouraged to do the prompting.
+All PRs must not conflict with [core design principles](../../space-station-14/core-design/design-principles.md). 
+Additionally, the PR author should describe how the changes they made fit into the game.
+Sufficiently large PRs may also warrant a **Design Doc** detailing the broader scope and purpose of their changes.
 
-- All non-gamebreaking revert PRs are **required** to undergo a Maintainer Discussion *before* being created. If there is a PR that you would like to be reverted, please *manually create a thread in maint-reviews with the format REVERT-PRNumber and ping the relevant work group Maintainers.* Discussions should reach a resolution within 48 hours, if discussions have stalled notify the Game Director or a PM.
-**This does not apply for reverts made to the staging branch during the review period**, as there will be a discussion thread for the release.
+All PRs that include **breaking changes** (*e.g. modifying public APIs, moving code into another namespace, changing a prototype ID, etc.*) should contain a "breaking changes" section.
+This consists of a summary of the changes as well as how to apply them.
+After the PR is merged, the maintainer should format a message containing the PR link, the entirety of the "breaking changes" section, and a ping for `@contrib-notification` and post it into the `#codebase-changes` channel of the Discord.
 
-- When closing a PR you are **required** to give a reason explaining why the PR is being closed, and if applicable this should include an alternative solution or approach. Stale PRs may just be closed for the reason of being stale, as described under the Requirements section. 
+All PRs must complete fill out the Github PR Template to a satisfactory level.
+This includes filling out all applicable sections and attaching media when necessary.
 
-- When reviewing a PR use *constructive* language and avoid referring negatively to the author's ability. Strong but fair criticism of code/decisions are fine, personal criticisms are not.
+## Decision Policy
+2 Maintainers are required to sign off on decisions about a PR.
+One of the maintainers is allowed to be an author of the PR, but the other should be uninvolved.
```

Comment:
> uninvolved

This seems undefined. What counts as uninvolved?
My interpretation of this would be "Multiple Maintainers may be an author of a PR, but only one can count for a sign off"

Reply:
The goal of this is to try and ensure that at least one maintainer reviewing a PR is not directly involved or advising it. It's fine if a maintainer guides and helps a PR and then reviews it, but i think it's compromising to the integrity of the review to have a group of maintainers assist and design a PR before merging it themselves.

in any case i've updated the wording.